funnel/templates/scan_badge.html.jinja2

```diff
@@ -25,7 +25,7 @@
             <p class="mui--text-danger mui--text-subhead page-content">{{ error }}</p>
           {{/if}}
           <video id="qrreader" class="scanner-wrapper__camera" playsinline autoplay></video>
-          {{#if video}}<svg class="fa5-icon scanner-wrapper__badge--icon" aria-hidden="true" role="img"><use xlink:href="{{ svgIconUrl }}#expand"></use></svg> {{/if}}
+          {{#if video}}<faSvg icon='expand' basline=false cssclass='scanner-wrapper__badge--icon'></faSvg> {{/if}}
```

Comment:
I think I saw `<fa-icon>` elsewhere. What's the difference?

Reply:
Changed for both vue and ractive component to faicon and user_avatar